### PR TITLE
ms965: add new copy role

### PR DESCRIPTION
**Pull request by MingtaoSun**
@scoen @terenceingram

### CODE REVIEW

File: src/amberdb/enums/CopyRole.java [hunk 1]
```diff
@@ -58,7 +58,9 @@
     THUMBNAIL_COPY("t", "Thumbnail",43),
     TIME_CODED_SUMMARY("sc", "Time coded Summary", "Yes",44),
     TIME_CODED_TRANSCRIPT_COPY("tc", "Time coded transcript", "Yes",45),
-    VIEW_COPY("v", "View",46);
+    VIEW_COPY("v", "View",46),
+    IMAGE_PACKAGE("ip", "Image Package",47);
```
Comment (terenceingram) [inline src/amberdb/enums/CopyRole.java]: Just wondering what the numbers are? 46, 47 etc. Is that the id in the database table? If that is the case then this enum is just horrible.

Comment (scoen) [inline src/amberdb/enums/CopyRole.java]: agreed. I suspect it was added as an ordering mechanism by someone :(

Comment (terenceingram) [inline src/amberdb/enums/CopyRole.java]: Very sad. Might add it to the list to fix. Pity we didn't pick it up in a code review :(

Comment (MingtaoSun) [inline src/amberdb/enums/CopyRole.java]: for ordering purpose. I don't think we store copy roles as reference data in the database.

Comment (scoen) [inline src/amberdb/enums/CopyRole.java]: Yeah, i think the problem with this approach is that everything after the first 4 or so entries everything else is alphabetical, so to add IMAGE_PACKAGE entry in the correct place (alphabetically) you'd have to renumber everything after it which would be a gigantic pita. So Image package will appear at the end of the list now. Sad face.

Comment (AaronCMuller) [inline src/amberdb/enums/CopyRole.java]: This is an ordering mechanism to ensure copy roles are shown in the correct order.

You can possibly get around this by externalising the order to a Map or paired list of some kind, but even if you do that you still have the same problem if you want to insert something with a similar order in the middle.

If it is indeed only the first four things that matter, then yes ... this should probably be refactored.

**Pull request merged**